theme-utils.mjs

```diff
@@ -1014,66 +1029,64 @@ Subscribers:
 }
 
 /*
- Create a Phabricator diff with the given message based on the contents currently in the sandbox.
- Open the phabricator diff in your browser.
- Provide the URL of the phabricator diff.
+ Create a GitHub pull request with the given message based on the contents currently in the sandbox.
+ Open the GitHub pull request in your browser.
+ Provide the URL of the GitHub pull request.
 */
-async function createPhabricatorDiff(commitMessage) {
+async function createGithubPR(commitMessage) {
 
-	console.log('creating Phabricator Diff');
+	console.log('Creating GitHub Pull Request');
 
 	let result = await executeOnSandbox(`
 		cd ${sandboxPublicThemesFolder};
 		git branch -D deploy
 		git checkout -b deploy
 		git add --all
 		git commit -m "${commitMessage}"
-		arc diff --create --verbatim
+		gh pr create --fill
```

Comment:
Will need to add `--head deploy` here to tell it to push to the `deploy` branch, otherwise it will give an interactive prompt for whether to push to `Automattic/wpcom-themes` or whether to create a fork and push there.

This is where gh diverges from arc. arc doesn't care that there are multiple diffs all created from a `deploy` branch, whereas gh will error out if there are two PRs trying to merge `deploy` with `trunk`. I think this behaviour is fine though, it makes sense to stop and evaluate the existing open PR.